xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/delete.vm

```diff
@@ -135,7 +135,10 @@
   <div class="panel $!panelClass">
     <div class="panel-heading">
       $heading
-      <div class="pull-right"><a class="panel-collapse-carret" role="button" data-toggle="collapse" href="#$panelId" aria-expanded="false" aria-controls="$panelId"><span class="caret"></span></a></div>
+      <div class="pull-right"><a class="panel-collapse-carret" role="button" data-toggle="collapse" href="#$panelId" aria-expanded="false" aria-controls="$panelId">
+        <span class="caret"></span>
+        <label class='sr-only'>$escapetool.xml($services.localization.render('core.delete.affectChildren.dropDown.label'))</label>
```

Comment:
The use of the `<label>`-element is invalid here as `<a>`-elements must not have interactive content as children, and a `<label>` is interactive content.

Reply:
Fixed in https://github.com/xwiki/xwiki-platform/pull/2215/commits/bb81586d9c450d8dae3644557c480331c3370fe5